libs/sound/include/OCPN_Sound.h

```diff
@@ -94,10 +94,9 @@ class OcpnSound
          * not playing. Returns true if stopping is supported, else false.
          */
         virtual bool Stop() = 0;
-
+        virtual bool Close() { return true; };
```

Comment:
Could this have a comment explaining the semantics. E.g. what to do in Close vs destructor, and what is the return value used for? If it is not used (and errors are logged and the return value ignored) why not just return void?

Reply:
... this is only used and overwritten in Portaudio. else it has no effect.